### DISCUSSION TITLE
Ex. 1: "Perfect" answer for "xy treatment"

### DISCUSSION THREAD

**Discussion by Sejmou** [sebastian-hofstaetter/teaching#62] [Advanced IR: Exercise Q&A] [answered]
Hi again,

I just got a result for a query of type "xy treatment" where the result discussed treatment of xy under certain conditions (low resources). Would this be considered a partial match, as it doesn't discuss the general approach for treating xy, or should I mark it as "perfect"?

Sorry if I am being nitpicky, but again, I just want to make sure the annotation task is done correctly.

Best,
Samo

**Comment by sebastian-hofstaetter** [comment, marked as answer]
Hi,

I would mark this as perfect. There can be multiple perfect answers to a given query. 

Best,
Sebastian
